test/PrintAsObjC/extensions.swift

```diff
@@ -95,8 +118,7 @@ extension GenericClass {
 class NotObjC {}
 extension NotObjC {}
 
-// CHECK-LABEL: @interface NSObject (SWIFT_EXTENSION(extensions))
-// CHECK-NEXT: @end
+// NEGATIVE-NOT: @interface NSObject (SWIFT_EXTENSION(extensions))
 // NEGATIVE-NOT: @interface NSObject{{$}}
 // NEGATIVE-NOT: @class NSObject
 extension NSObject {}
```

Comment:
The CHECK and NEGATIVE lines are being checked in two different passes, so it's not going to be a problem in practice. I'm happy to have them reordered, though; that would match the cases where we *do* expect the extended class to be defined first.